main.py

```diff
@@ -118,7 +124,7 @@ def handle_event(event: Event):
     metadata: dict = extract_metadata(path)
 
     # Build a mediahaven sidecar with extracted xmls
-    sidecar = create_sidecar(metadata)
+    sidecar = create_sidecar(path, metadata)
```

Comment:
Is `path` of type `Path` or is it just `str`?

Reply:
It is `str` but it was incorrectly typed in the function.